week5/card_validation.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+'''
+Write a program that prompts the user to enter
+a credit card number as an integer.
+Display whether the number is valid or invalid.
+
+1. Double every second digit from right to left.
+If doubling of a digit results in a two­digit number,
+add up the two digits to get a single­digit number
+
+2. Now add all single­digit numbers from Step 1.
+4 + 4 + 8 + 2 + 3 + 1 + 7 + 8 = 37
+
+3. Add all digits in the odd places from right to
+left in the card number.
+6 + 6 + 0 + 8 + 0 + 7 + 8 + 3 = 38
+
+4. Sum the results from Steps 2 and 3.
+37 + 38 = 75
+
+5. If the result from Step 4 is divisible by 10
+the card number is valid; otherwise, it is invalid.
+For example, the number 4388576018402626 is invalid
+but the number 4388576018410707 is valid.
+'''
+
+#Create a matrix of numbers from card number string
+def numberToMatrix(number):
+    return [ int(char) for char in number ]  
```

Comment:
According to the [pep-0257](https://www.python.org/dev/peps/pep-0257/) (about docstring in python), you should put the docstring after the `def` line of the function such as:
```
def numberToMatrix(number):
    """Create a matrix of numbers from card number string"""
        return [ int(char) for char in number ] 
```
That way it can be use in the __doc__ attribute of your function and other software that will parse the doctstings to create doc.

Note: Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.

I encourage you to read the pep.